### DISCUSSION TITLE
Use alternative icons in YAML files

### DISCUSSION THREAD

**Discussion by cbrnr** [quarto-dev/quarto-cli#2776] [Q&A] [answered]
I would like to use icons not contained in the standard [Bootstrap icon set](https://icons.getbootstrap.com/) in my `_quarto.yml` file (i.e. I want to put an [Academicon](https://jpswalsh.github.io/academicons/), e.g. the orcid one, in my top navigation bar). The Academicon extension works only for `.qmd` files (https://github.com/schochastics/academicons/issues/2). Is this possible somehow?

**Comment by mcanouil** [comment, marked as answer]
See answer of #2402

Example: https://github.com/RLille/rlille.fr/blob/main/_quarto.yml

**Reply by cbrnr** [reply]
Thank you!

**Reply by mcanouil** [reply]
You are welcome 😉
